department_project/department_app/service/department_service.py

```diff
@@ -0,0 +1,100 @@
+"""Module for working with departments in site"""
+import requests
+from django.http import HttpResponseRedirect
+from django.views.generic import DeleteView, CreateView, ListView, UpdateView
+from rest_framework.reverse import reverse
+
+from department_app.models import Department
+
+
+class DepartmentTemplate(ListView):
+    """ Template for view the list department"""
+    model = Department
+    fields = '__all__'
+    object_list = None
```

Comment:
These are the redundant lines.

Reply:
object_list - removed. 
'fields' must be for model.